### ISSUE TITLE
Fix Invoke App From Background Android 10

### ISSUE THREAD

**Issue by naufanituharish** [bkdev98/react-native-incoming-call#15]
Hello there,, thanks for the library,, it working awesome for incoming call screen,.
I found solution for your problems, to invoke incoming call screen from locked screen in android 10, you could use full screen intent, i use it with voximplant library and it works with locked screen and killed state app,
You can follow this:
https://github.com/voximplant/android-demos/tree/master/audiocall
and using the notificationhelper.java in this link:
https://github.com/voximplant/android-demos/blob/master/audiocall/src/main/java/com/voximplant/demos/audiocall/utils/NotificationHelper.java
And also use foreground service check in those repo,.
Since I'm not android expert and new in RN so I could not make PR for this case,.

Hope this information can help you,.

Regards,
Naufan harish

**Comment by kimho** [comment]
@naufanituharish Could you please show me your code?

**Comment by bkdev98** [comment]
Hi @naufanituharish, I try using your fork repo and get an exception ForegroundCheck is not initialized. Did you add additional code in your project?

**Comment by bkdev98** [comment]
I see in voximplant code call ForegroundCheck.init(this) but I'm not sure where to put it yet 😅

**Comment by Bilal112** [comment]
it's working you to get permission or overlay

**Comment by bkdev98** [comment]
Yes I'm also using it. But actually, some Xiaomi/Oppo devices need additional "Display pop-up windows while running in the background" permission to make it work. So I'm finding some replacement solutions.

Here for detail https://stackoverflow.com/questions/59645936/displaying-popup-windows-while-running-in-the-background

**Comment by Bilal112** [comment]
I also using the Xiaomi set and its working

**Comment by naufanituharish** [comment]
> Hi @naufanituharish, I try using your fork repo and get an exception ForegroundCheck is not initialized. Did you add additional code in your project?

Hi @bkdev98 sorry for very late reply currently I'm working on other project, have you solved it?? if not, give me a few days to review my code,. because long time not touch my previous work. need to know the flow,. 😅😅

**Comment by bkdev98** [comment]
Yeah I think I'm fine with the rn-android-overlay-permission solution for now 😄

**Comment by AliSaeed090** [comment]
@bkdev98  @Bilal112 
I am using rn-android-overlay-permission , its working fine when app is in background  state, Not working when app is quit or killed. any solution when app is in killed or quit state?

**Comment by fukemy** [comment]
we need provide autostart permission first, this lib does not
